src/backend/InvenTree/part/models.py

```diff
@@ -467,6 +467,11 @@ class MPTTMeta:
         # For legacy reasons the 'variant_of' field is used to indicate the MPTT parent
         parent_attr = 'variant_of'
 
+    @property
+    def parent(self):
+        """Return the template part for this Part instance."""
+        return self.variant_of
```

Comment:
Q: This a change in behavior, right?

Reply:
That was only a helper func in there for testing. I've removed it now.